open_humans/templates/pages/home.html

```diff
@@ -1,87 +1,21 @@
 {% extends 'base.html' %}
 
 {% load static from staticfiles %}
+{% load utilities %}
 
-{% block head_title %}Open Humans{% endblock %}
-{% block head_title_suffix %}{% endblock %}
+{% block head_title %}Home{% endblock %}
 
-{% block body_main %}
-
-{% if user.is_authenticated %}
-<div class="background-horizontal-block">
-  <div class="container">
-    <div class="panel-over-full-background col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
-      <div class="row">
-        <div class="col-xs-12">
-          {% if messages %}
-            {% for message in messages %}
-            <div{% if message.tags %} class="message {{ message.tags }}"{% endif %}>{{ message }}</div>
-            {% endfor %}
-          {% endif %}
-        </div>
-      </div>
-
-      <div class="row" style="padding-bottom:15px;">
-        <div class="col-md-12 text-center">
-          <h4>Welcome, {{ user.username }}!</h4>
-
-          <p>Where would you like to go from here?</p>
-
-          <div class="row-fluid">
-            <hr class="visible-xs-block" style="margin:20px;">
-
-            <div class="col-sm-4">
-              <p>
-                <a class="btn btn-primary btn-sm"
-                    href="{% url 'my-member-research-data' %}">
-                  Research data
-                </a>
-              </p>
-
-              <p>
-                Import and manage data from studies and activies.
-              </p>
-            </div>
-
-            <hr class="visible-xs-block" style="margin:20px;">
-
-            <div class="col-sm-4">
-              <p>
-                <a class="btn btn-primary btn-sm"
-                    href="{% url 'my-member-dashboard' %}">
-                  Profile page
-                </a>
-              </p>
-
-              <p>
-                Add a profile pic and tell people about yourself!
-              </p>
-            </div>
-
-            <hr class="visible-xs-block" style="margin:20px;">
-
-            <div class="col-sm-4">
-              <p>
-                <a class="btn btn-primary btn-sm"
-                    href="{% url 'my-member-settings' %}">
-                  Account settings
-                </a>
-              </p>
+{% block extra_js %}
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.0/imagesloaded.pkgd.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/2.2.2/isotope.pkgd.min.js"></script>
```

Comment:
I think we'll want to add these to package.json and include them locally so we can see when they become outdated (but this can happen later)